### ISSUE TITLE
8.2.3 seems to have broken front matter automatic suggestion

### ISSUE THREAD

**Issue by dreambrace** [tadashi-aikawa/obsidian-various-complements-plugin#235]
After 8.2.3, pressing space in the appropriate place no longer automatically triggers the suggestion window for available values in the front matter. It can only be triggered with the command/hotkey.

**Comment by tadashi-aikawa** [comment]
@dreambrace 
Could you tell me the reproduction steps in the sandbox vault?

**Comment by dreambrace** [comment]
No problem

1. Fresh install through community plugins, strategy: english only
2. Create a note with front matter:
```
---
foo: bar, baz
---
```

OR

```
---
foo:
  - bar
  - baz
---
```
3. Create another note with front matter:
```
---
foo:
---
```
OR
```
---
foo:
  -
---
```
When you press space after the colon in the first example or the dash in the second, the suggestion window should automatically pop up, but doesn't. You can open it manually in that spot, so I'm assuming it should trigger. In 8.2.2 and prior, it works that way.

Note:
- With the default strategy, typing the colon after the key also triggers the suggestion window, and pressing space then closes it. In 8.2.2 it stays open after pressing space. I assume it's the same behaviour, but it's easier to explain with the example above.

**Comment by tadashi-aikawa** [comment]
Thanks, @dreambrace :)  I could reproduce it 👍 

Maybe, the following responses caused this problem. I'll fix it in the next release.

> Suppress unnecessary completion trigger ([#221](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/issues/221))

**Comment by tadashi-aikawa** [comment]
@dreambrace 
Fix in [v8.3.2](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/releases/tag/8.3.2) 🚀 

> **Note**
> There are times when extra suggestions are displayed. It will improve it in a future release.

**Comment by dreambrace** [comment]
Works great! Your speed is unparalleled. Thank you for all the work on this great plugin :)
